back-end/routes/routes.js

```diff
@@ -22,4 +23,5 @@ module.exports = server => {
     server.route('/browse').get(fetchImages);
     server.route('/collectionAdd').post(collection);

```

Comment:
this `/collectionAdd` route looks like an extraneous route. Should it be deleted?